src/clj/game/cards/operations.clj

```diff
@@ -1360,7 +1360,11 @@
                                  (if (same-card? card installed-card)
                                    ((constantly false) (toast state :corp "Cannot score due to Mushin No Shin." "warning"))
                                    true)))
-                             (effect-completed state side eid))))}})
+                             (effect-completed state side eid))))}
+   :events [{:event :corp-turn-begins
+             :duration :until-corp-turn-begins
```

Comment:
This will persist (lol) past when the operation is trashed at the end of the run?

Reply:
I'm not sure I understood your question: operations are trashed immediately after resolving..

Reply:
`:events` entries are registered when an operation is played and before it resolves, and then after it's resolved they are removed. I think the `:duration` part means it won't be unregistered after resolution but I don't remember.

Reply:
All I can say is the updated test is now green :)